apps/transport/lib/transport_web/templates/page/index.html.eex

```diff
@@ -25,10 +25,10 @@
                     <h3 class="text-center"><%= dgettext("page-index", "Realtime public transit") %></h3>
                     <p><%= dngettext("page-index", "dataset", "datasets", @count_has_realtime) %></p>
                 </a>
-                <a class="tile" href="<%= dataset_path(@conn, :index, "tags[]": "bus") %>">
+                <a class="tile" href="<%= dataset_path(@conn, :index, ["tags[]": "bus", region: 14]) %>">
```

Comment:
not really satisfied about this. But I suppose it would require first to rethink the nationalist crap